test/map.jl

```diff
@@ -25,6 +25,11 @@ using Test
         fA = f.(A)
         @test eltype(fA) == N0f8
         @test fA == [N0f8(0), N0f8(0.4), N0f8(1)]
+
+        A = [-1.2,0.4,800.3]
+        clamp01!(A)
+        @test eltype(A) == Float64
```

Comment:
`A`'s eltype can't change. But of course this doesn't hurt anything, so no need to remove it.